src/content/app/browser/hooks/useGenomeBrowser.ts

```diff
@@ -155,23 +158,26 @@ const useGenomeBrowser = () => {
 
   const changeBrowserLocation = (locationData: {
     genomeId: string;
-    ensObjectId: string | null;
+    focusId?: string;
     chrLocation: ChrLocation;
   }) => {
     if (!genomeBrowser) {
       return;
     }
 
-    const { genomeId, chrLocation, ensObjectId } = locationData;
+    const { genomeId, chrLocation, focusId } = locationData;
+
+    const { objectId = null } = focusId ? parseFocusIdFromUrl(focusId) : {};
```

Comment:
So `useGenomeBrowser` sends to the genome browser only the stable id of a feature, not the whole "gene:stable_id" thing? Does this perhaps suggest that the focus  url parameter should have been parsed in `useBrowserRouting`, and that the `changeBrowserLocation` function should have received just the gene stable id in the `focusId` field?

Reply:
>So useGenomeBrowser sends to the genome browser only the stable id of a feature, not the whole "gene:stable_id" thing?
Yes, it just sends the stable id so it makes more sense to parse it in `useBrowserRouting` itself. That's done.